typescript/infra/config/environments/testnet4/chains.ts

```diff
@@ -18,6 +19,12 @@ export const chainMetadataOverrides: ChainMap<Partial<ChainMetadata>> = {
       gasPrice: 8 * 10 ** 9, // 8 gwei
     },
   },
+  kyvetestnet: {
+    transactionOverrides: {
+      gasPrice: '2.0',
+    },
+    protocol: ProtocolType.CosmosNative,
+  },
```

Comment:
manually overriding temporarily until https://github.com/hyperlane-xyz/hyperlane-registry/pull/731 is merged

(unable to merge yet because we're debugging an explorer package update that has to be complete before we can use the new protocol types https://github.com/hyperlane-xyz/hyperlane-explorer/pull/192)